### ISSUE TITLE
iron golem visual bug

### ISSUE THREAD

**Issue by titaniumman321** [Yesssssman/epicfightmod#550]
![2021-09-13_11 45 03](https://user-images.githubusercontent.com/84505406/133028048-d3b88372-efea-49ad-9edb-f54d03000d86.png)

the cracks appear above the actual model

**Comment by Yesssssman** [comment]
Is this still happens when the shaders are off?

**Comment by titaniumman321** [comment]
I am not using shaders at all

**Comment by Yesssssman** [comment]
![2021-10-06_04 37 14](https://user-images.githubusercontent.com/79469058/136091286-3cf24acf-9793-4040-9059-fdc6ac18b23d.png)

I captured this screenshot in 5.1.0. But I didn't change any code regarding the iron golem. Could you tell me if this problem still happens?

**Comment by titaniumman321** [comment]
yes the bug is still happening even on the latest version , plus shield animations are not working on the left hand slot
